lib/galaxy/dependencies/pinned-requirements.txt

```diff
@@ -1,11 +1,11 @@
 # packages with C extensions
+numpy==1.9.2
 bx-python==0.7.3
```

Comment:
Can you also please document this in a short comment, so we don't regress in the future?